src/libaktualizr/package_manager/packagemanagerconfig.h

```diff
@@ -7,15 +7,20 @@
 
 #include "utilities/config_utils.h"
 
-enum class PackageManager { kNone = 0, kOstree, kDebian, kAndroid, kOstreeDockerApp };
-std::ostream& operator<<(std::ostream& os, PackageManager pm);
+#define PACKAGE_MANAGER_NONE "none"
+#define PACKAGE_MANAGER_OSTREE "ostree"
+#define PACKAGE_MANAGER_DEBIAN "debian"
+#define PACKAGE_MANAGER_ANDROID "android"
+#define PACKAGE_MANAGER_OSTREEDOCKERAPP "ostree+docker-app"
```

Comment:
This looks pretty cool! The only thing I don't like is that we still have to define the strings here directly in libaktualizr.

Reply:
Actually, that should not be required. It's just a convenience for the package manager that are bundled into libaktualizr (all of them right now). 

And from a quick look, even these could probably be moved to their corresponding headers.

Reply:
> And from a quick look, even these could probably be moved to their corresponding headers.

I realize it's convenient to have them grouped together, but conceptually, I think it makes sense to move them to their respective headers, since that's what actual custom implementations will have to do.